.github/workflows/autoware-base.yaml

```diff
@@ -46,6 +35,6 @@ jobs:
         with:
           target-image: autoware-base
           build-args: |
-            *.platform=linux/arm64
+            *.platform=linux/amd64,linux/arm64
```

Comment:
I thought we [separated this job to avoid disk exhaustion](https://github.com/autowarefoundation/autoware/pull/5487/files), do you want to merge it back again?

Reply:
It was discovered that if we push images with the same tag name at a different time, only the one pushed later will be adopted, and it will not become multi-architecture images.
https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/

It is believed that disk exhaustion will not occur if the `free-disk-space` action is executed even on an `ubuntu-22.04` runner.
https://github.com/youtalk/autoware/actions/runs/12119766196